src/ui/widgets/Symbol/symbol.tsx

```diff
@@ -102,6 +102,9 @@ export const SymbolComponent = (props: SymbolComponentProps): JSX.Element => {
     }
   }
 
+  const cursor =
+    props.actions && props.actions.actions.length > 0 ? "pointer" : "auto";
+
```

Comment:
nice touch, didn't know you could do that, I think could be simplified to
`props.actions?.actions?.length > 0 ? "pointer": "auto"` though

Reply:
Ah I assume the reason you didn't do this was because of the typescript error of `this could be undefined`.